src/panels/lovelace/editor/config-elements/hui-generic-entity-row-editor.ts

```diff
@@ -0,0 +1,173 @@
+import "@polymer/paper-input/paper-input";
+import {
+  CSSResult,
+  customElement,
+  html,
+  internalProperty,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+import { assert } from "superstruct";
+import { fireEvent } from "../../../../common/dom/fire_event";
+import { computeDomain } from "../../../../common/entity/compute_domain";
+import { stateIcon } from "../../../../common/entity/state_icon";
+import "../../../../components/ha-formfield";
+import "../../../../components/ha-icon-input";
+import "../../../../components/ha-switch";
+import { HomeAssistant } from "../../../../types";
+import { EntitiesCardEntityConfig } from "../../cards/types";
+import "../../components/hui-action-editor";
+import "../../components/hui-entity-editor";
+import "../../components/hui-theme-select-editor";
+import { LovelaceRowEditor } from "../../types";
+import {
+  EditorTarget,
+  entitiesConfigStruct,
+  EntitiesEditorEvent,
+} from "../types";
+import { configElementStyle } from "./config-elements-style";
+
+const SecondaryInfoValues: { [key: string]: { domains?: Set<string> } } = {
+  "entity-id": {},
+  "last-changed": {},
+  "last-triggered": { domains: new Set(["automation", "script"]) },
+  position: { domains: new Set(["cover"]) },
+  "tilt-position": { domains: new Set(["cover"]) },
+  brightness: { domains: new Set(["light"]) },
```

Comment:
I actually did some research lately and for arrays < 2500 elements, a `array.include` is way fast than a `set.has` 😄